python/paddle/fluid/tests/unittests/op_test.py

```diff
@@ -371,6 +371,20 @@ def check_output_with_place(self,
                                 no_check_set=None,
                                 equal_nan=False,
                                 check_dygraph=False):
+        self._check_output_with_place(place, atol, no_check_set, equal_nan,
+                                      check_dygraph)
+        if self._is_gc_test_enabled():
+            self._enable_gc(True)
+            self._check_output_with_place(place, atol, no_check_set, equal_nan,
+                                          check_dygraph)
+            self._enable_gc(False)
```

Comment:
If ` self._is_gc_test_enabled()` is true, the `self._check_output_with_place` will be called two times, is it necessary?

Reply:
For safety check, though I have not figured out any case that the calculating result would be different if using gc or not.